src/helpers.js

```diff
@@ -56,3 +56,44 @@ exports.getTopic = (attributes, slots) => {
         default: return (slots.topic && slots.topic.value) ? slots.topic.value : null;
     }
 };
+
+exports.getPodcastDirective = (podcastUrl) => {
```

Comment:
just a style preference thing, but I wouldn't have defined the object outside of this function, there's no need